libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/view/MapboxManeuverView.kt

```diff
@@ -231,10 +231,8 @@ class MapboxManeuverView : ConstraintLayout {
     )
     fun renderManeuverShields(shieldMap: Map<String, RoadShield?>) {
         val shields = shieldMap.values.filterNotNull().map(RoadShield::toRouteShield)
-        synchronized(routeShields) {
```

Comment:
Can you do the same for `MapboxRoadNameView`?

Reply:
yeah, missed that one.